input/parser_test.go

```diff
@@ -252,7 +252,18 @@ func Test_Parsing(t *testing.T) {
 			if !ok {
 				t.Error("\n\nParser should succeed.\n")
 			}
-			if !reflect.DeepEqual(p.Result, tt.expected) {
+			ckeckEquality := func() bool {
+				for k, v := range p.Result {
```

Comment:
For some reason, I this equality test (as well as `reflect.DeepEqual()` does not work.